crates/nargo_cli/tests/test_data_ssa_refactor/simple_comparison/src/main.nr

```diff
@@ -0,0 +1,6 @@
+// Tests a very simple program.
+// 
+// The features being tested is comparison
+fn main(x : Field, y : Field)  {
+    assert(x as u32 < y as u32);
```

Comment:
Note this is not testing >= because we need to process not for this. I'm leaving that for another PR as this PR requires quite a bit of architecture changes